### DISCUSSION TITLE
Introduce-yourself!

### DISCUSSION THREAD

**Discussion by prasanna-muppidwar** [ML-Nagpur/Cool-Notebooks#1] [General]
Hey! Im Prasanna

**Comment by AayushPaigwar** [comment]
Hello Prasanna,
Aayush Paigwar this side!👋🏻

**Comment by yagyesh-bobde** [comment]
Hiii!! I am Yagyesh Bobde. I am an ML enthusiast. Would love to connect with you guys.

**Comment by Atharva963** [comment]
Hello! Atharva here...

**Comment by AadityaThantharate** [comment]
Hello !.....Aaditya here🙂
